userspace/libsinsp/container.cpp

```diff
@@ -160,6 +160,7 @@ string sinsp_container_manager::container_to_json(const sinsp_container_info& co
 	container["privileged"] = container_info.m_privileged;
 	container["is_pod_sandbox"] = container_info.m_is_pod_sandbox;
 	container["lookup_state"] = static_cast<int>(container_info.m_lookup_state);
+	container["created_time"] = (Json::Value::Int64) container_info.m_created_time;
```

Comment:
why is the cast:
1. C-style (not a static_cast<>)
2. not to an int64_t
?

Reply:
The `Json::Value` library we are using only has defined `Int64` 
```
class JSON_API Value {
  ...
#if defined(JSON_HAS_INT64)
  typedef Json::UInt64 UInt64;
  typedef Json::Int64 Int64;
...
```

This `int64` in turn is `typedef long long int Int64;` 
Also, for the c-style cast I just copied what similar fields in this function were doing.

Reply:
Let me `static_cast` it.

Reply:
Fixed.